backend/app/cmd/server.go

```diff
@@ -735,6 +773,7 @@ func (s *ServerCommand) makeAuthenticator(ds *service.DataStore, avas avatar.Sto
 			}
 			c.User.SetAdmin(ds.IsAdmin(c.Audience, c.User.ID))
 			c.User.SetBoolAttr("blocked", ds.IsBlocked(c.Audience, c.User.ID))
+			c.User.Email, _ = ds.GetStringUserDetail(store.Locator{SiteID: c.Audience}, c.User.ID, engine.UserEmail)
```

Comment:
If the ignored param is error I get why you can't return it, but at list log warn will be helpful in case if this is "not found" but smth failed for real, i.e. permission error, corrupted bolt file and so on